plugin/forward/metrics.go

```diff
@@ -22,3 +46,17 @@ var (
 		Help:      "Counter of the number of queries rejected because the concurrent queries were at maximum.",
 	})
 )
+
+func recordReqMetrics(addr string, start time.Time, res *dns.Msg) {
+	RequestCount.WithLabelValues(addr).Add(1)
+
+	if res != nil {
+		// Record metrics
+		rc, ok := dns.RcodeToString[res.Rcode]
+		if !ok {
+			rc = strconv.Itoa(res.Rcode)
+		}
+		RcodeCount.WithLabelValues(addr, rc).Add(1)
```

Comment:
Similar here, we don't actually need `RcodeCount` since we get the same values from the `RequestDuration` metric `_count` data.

We can remove this and mark it as a breaking change.

Reply:
I've removed the duplicate metrics from `forward` completely.